examples/controls/src/app.js

```diff
@@ -3,15 +3,11 @@ import React, {Component} from 'react';
 import {render} from 'react-dom';
 import MapGL, {Marker, Popup, NavigationControl} from 'react-map-gl';
 
-import CityPin from './components/city-pin';
+import CityPin from './city-pin';
 
-import CITIES from './data/cities.json';
+import CITIES from './cities.json';
 
-import './app.css';
-import 'mapbox-gl/dist/mapbox-gl.css';
-
-const token = process.env.MAPBOX_ACCESS_TOKEN || // eslint-disable-line
-  'Set MAPBOX_ACCESS_TOKEN environment variable or put your token here.';
+const token = process.env.MapboxAccessToken; // eslint-disable-line
 
 if (!token) {
   throw new Error('Please specify a valid mapbox token');
```

Comment:
I believe the environment variable is replaced by webpack at compile time. Not sure how it will work without requiring the user to add additional configs to their bundling setup.